stage0/layout.ld

```diff
@@ -150,6 +150,33 @@ SECTIONS {
     gdt_desc_offset = ADDR(.rodata.gdt_desc) & 0xFFFF;
     idt_desc_offset = ADDR(.rodata.idt_desc) & 0xFFFF;
 
+    /* GUIDed tables have to *end* at 0x20 from the end of the file. */
+    .guid_tables TOP - 0x20 - 0x28 : {
+        HIDDEN(tables_start = .);
+        /* SEV-ES reset block: uint32 addr, uint16 size, 16-byte GUID */
+        HIDDEN(sev_es_reset_block_start = .);
+        LONG(0xDEADBEEF) /* Placeholder values */
+        SHORT(sev_es_reset_block_end - sev_es_reset_block_start)
+        LONG(0x00f771de)
+        SHORT(0x1a7e)
+        SHORT(0x4fcb)
+        SHORT(0x0e89)
+        BYTE(0x68) BYTE(0xc7) BYTE(0x7e) BYTE(0x2f) BYTE(0xb4) BYTE(0x4e)
```

Comment:
Is there a reference somewhere (e.g. the OVMF repository) where the GUIDs are defined so that we can add to explain where these magic values come from?

Reply:
I've added a couple of links.